Guide/src/reference/architecture/openhcl.md

```diff
@@ -7,71 +7,60 @@
 
 * * *
 
-> This page is under construction
-
 ## Overview
 
-The following diagram offers a brief, high-level overview of the OpenHCL
-Architecture.
+OpenHCL is a paravisor execution environment that runs within the guest partition of a virtual machine. It provides virtualization services to the guest OS from within the guest itself, rather than from the host.
```

Comment:
```suggestion
OpenHCL is a paravisor execution environment that runs within the guest partition of a virtual machine. It provides virtualization services to the guest OS from within the guest partition itself, rather than from the host as is traditionally done.
```

Reply:
or something to really distinguish ourselves from "normal" vmms